ComplementaryData/physiology/biomassComposition_Cofactor_Ion.tsv

```diff
@@ -0,0 +1,20 @@
+s_3880[c]	Ca(2+) 	2.17E-04	40.08	ion	DOI: 10.22161/ijeab/2.2.2
+s_3778[c]	chloride 	2.55E-03	35.45	ion	DOI: 10.22161/ijeab/2.2.2
+s_4019[c]	Cu2(+) 	6.59E-04	63.55	ion	DOI: 10.22161/ijeab/2.2.2
+s_0924[c]	iron(2+) 	3.04E-05	55.84	ion	DOI: 10.22161/ijeab/2.2.2
+s_3801[c]	Mn(2+) 	2.73E-03	54.94	ion	DOI: 10.22161/ijeab/2.2.2
+s_3822[c]	Zn(2+) 	7.48E-04	65.39	ion	DOI: 10.22161/ijeab/2.2.2
+s_1373[c]	potassium 	3.63E-03	39.1	ion	DOI: 10.22161/ijeab/2.2.2
+s_1437[c]	sodium 	3.97E-03	22.99	ion	DOI: 10.22161/ijeab/2.2.2
+s_4013[c]	Mg(2+) 	0.001242543	24.3	ion	DOI: 10.22161/ijeab/2.2.2
+s_1467[c]	sulphate	0.02	96.06	ion	Forster2003
+s_1203[c]	NADH 	1.5E-04	663.43	cofactor	DOI: 10.1016/j.ymben.2013.11.005; Supplementary file
+s_1198[c]	NAD 	2.65E-03	662.43	cofactor	DOI: 10.1016/j.ymben.2013.11.005; Supplementary file
+s_1212[c]	NADPH 	2.7E-03	745.4209	cofactor	DOI: 10.1016/j.ymben.2013.11.005; Supplementary file
+s_1207[c]	NADP(+) 	5.7E-04	744.4129	cofactor	DOI: 10.1016/j.ymben.2013.11.005; Supplementary file
+s_0529[c]	coenzyme A 	1.90E-04	767.5341	cofactor	DOI: 10.1016/j.ymben.2013.11.005; Supplementary file
+s_1487[c]	THF 	6.34E-05	445.4292	cofactor	DOI: 10.1021/jf048083g
+s_1475[c]	TDP 	0.000012	425.3144	cofactor	DOI:10.1002/jib.293

```

Comment:
please also include where in the reference the data is (e.g. figure/table X, or page X/paragraph Y), as I could not find the source in this paper (has 15 figures). To be consistent, please include this info for all metabolites in this table

Reply:
Some metablites's evidence is from YMDB database (http://www.ymdb.ca)

Reply:
@hongzhonglu then the full link to the specific metabolite should be provided, e.g. http://www.ymdb.ca/compounds/YMDB00001

Reply:
@BenjaSanchez When I help Feiran to do the commit, after I use scaleBioMass function, when I further use the function' model=loadYeastModel ', the followed error is reported:
>> model=loadYeastModel
No model returned.
Reference to non-existent field 'annotation'.

Error in readSBML (line 34)
if ~isempty(modelSBML.annotation)

Error in readCbModel (line 208)
        model = readSBML(fileName,defaultBound);

Error in loadYeastModel (line 12)
model = readCbModel('../ModelFiles/xml/yeastGEM.xml');

Do you know the reason? I have updated the RAVEN AND cobra to the latest version.

Reply:
I do not know the reason, but this behavior does not happen in my PC, neither in `devel` nor in this branch. It sounds like there could be a problem with your COBRA, have you ran `initCobraToolbox` beforehand? Otherwise, consider re-installing libSBML + COBRA + checking any added paths you have in your Matlab session, as the latter might introduce unexpected behaviors (see https://github.com/SysBioChalmers/yeast-GEM/pull/174#discussion_r230785011)

Reply:
Thanks for your help! Will check it today.

Reply:
@BenjaSanchez  I update cobra. LibSBML is also put in the matlab path. the readCbModel can be used before using scaleBiomass function. Sadly, after running scaleBiomass function, readCbModel will return the same error.

Reply:
@hongzhonglu then that must be an error that is being introduced by the new changes that are not yet in this PR, as otherwise `readCbModel` works well in this branch. Look into the new changes you are making (as I cannot see them yet here) and find what is causing problems. You may also ask the COBRA developers about the specific error you are getting, they might be able to help you.

Reply:
@BenjaSanchez  Should I pull the changes firstly? Then you can compare the differences

Reply:
@hongzhonglu great that you found it, can you correct it in the PR? Also, the original point of this comment has not been addressed yet: the figure/table or page/paragraph where the data was obtained in each reference should be included as additional information, not just the DOI. This to aid the reader find the original data. As an example, I could not find the source of the data for TDP in https://doi.org/DOI:10.1002/jib.293

Reply:
@BenjaSanchez I have corrected the rxn name already, which should be corrected from the original xml file. The detailed information need further input from @feiranl.

Reply:
@hongzhonglu however your [latest commit](https://github.com/SysBioChalmers/yeast-GEM/pull/174/commits/fae399985f233e13ac39cbd3fc5bda06fe32d22c) does not show that change

Reply:
@BenjaSanchez I omit to record this step.

Reply:
@hongzhonglu that name is actually non-compliant, as we should avoid greek letters. Will create a separate PR to solve that.